### ISSUE TITLE
Add Table of all Service Instances to OT Server Details view

### ISSUE THREAD

**Issue by bigal91** [CloudCycle2/CSAR_Repository#330]
This table has to contain all Service Instances, that are running on the current OT Server (get with containerapi). 

For every Instance there has to be a link to the CSAR, from which it was instantiated.
The names of those instances are randomly numbered in something like the following manner:
"NAME_OF_CSAR"_XX, where XX is a random number... This makes it impossible to map a specific CSAR-FILE to a Cloud-Instance. We can only compare the CSAR Name / Service Template Name ("NAME_OF_CSAR") with the Cloud-Instance ("Servie-Instance") name. If its the same our system may assume a connection between those - the CSAR and the Service-Instance.

They have to contain Links for the MANAGEMENT and TERMINATION Plans (which we can find out as well over the containerapi... by knowing the "NAME_OF_CSAR"/PLANS/* ... something like this)

Additionally there should be a general link to the Instance-Data API (containerapi) of the OT-Server.

**Comment by eiselems** [comment]
TLDR:
Query ServiceInstances from InstanceData-API
Show in List:
* links to ServiceInstance-Detail View
* way to invoke Plans (everything but BUILD)

**Comment by przytads** [comment]
#301
